scripts/run.sh

```diff
@@ -0,0 +1,45 @@
+## Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.## 
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0 
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## 
+
+#! /usr/bin/env bash
+set +x
+
+export DEBIAN_FRONTEND=noninteractive
+
+if [ -n "$INPUT_HOLODECK_CONFIG" ]; then
```

Comment:
How are the variables in `action.yml` mapped to these envvars?

Reply:
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs